peering/models.py

```diff
@@ -1357,10 +1365,10 @@ def get_napalm_device(self):
             self.logger.debug('found napalm driver "%s"', self.platform)
             return driver(
                 hostname=self.hostname,
-                username=settings.NAPALM_USERNAME,
-                password=settings.NAPALM_PASSWORD,
-                timeout=settings.NAPALM_TIMEOUT,
-                optional_args=settings.NAPALM_ARGS,
+                username=self.napalm_username,
```

Comment:
Continue to use the existing configuration options if the user does not define a per-router NAPALM configuration.